arches/app/etl_modules/branch_excel_exporter.py

```diff
@@ -45,7 +45,7 @@ def __init__(self, request=None, loadid=None):
         self.loadid = loadid if loadid else None
 
 
-    def run_export_task(self, load_id, graph_id, graph_name, resource_ids):
+    def run_export_task(self, load_id, graph_id, graph_name, resource_ids, *args, **kwargs):
```

Comment:
I take it the reason this is dynamic is so that the BranchExcelExporter doesn't have to implement the new argument. If that's the reason, we could also just let that implementation accept a default argument value of None (or UUIDs, since that's what it seems to do) and then raise NotImplementedError if it got labels. Not important, but there's something nice about the explicitness IMO. Up to you!